charts/minecraft/values.yaml

```diff
@@ -180,6 +180,8 @@ minecraftServer:
     existingSecret:
     secretKey: rcon-password
     serviceType: LoadBalancer
```

Comment:
I hadn't noticed before the default is LoadBalancer. I think I'll change that to ClusterIP after merging your PR.

Reply:
yes this makes sense

Reply:
Agreed, `ClusterIP` is a safer default to use IMO.  Not everyone is going to have a cluster that can handle `LoadBalancer`.

Reply:
I've switched to ClusterIP default for rcon